rules/jsll/src/rules/jsll-awa-init/jsll-awa-init.ts

```diff
@@ -57,7 +58,7 @@ const rule: IRuleBuilder = {
         });
 
         const validateScript = async (scriptParse: IScriptParse) => {
-            if (!hasJSLLScript) {
+            if (!hasJSLLScript || !isHead) {
```

Comment:
Shouldn't we validate the script even if it is not in the `head` ?

Reply:
If the init script is not in the head, a report is raise during`enterBody` with the message of `JSLL is not initialized with "awa.init(config)" function. Initialization script should be placed immediately after JSLL script.` According to the spec, the JSLL init script should be `in separate script tag immediately following the JSLL script`. @molant Let me know if you think it should be handled differently.

Reply:
I think we should raise the error but still try to continue validating things so we can give as much information as possible.